lib/common.bash

```diff
@@ -185,3 +185,14 @@ get_pod_config_dir() {
 	pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
 	info "k8s configured to use runtimeclass"
 }
+
+# Uses crictl to pull a container image passed in $1.
+# If crictl is not found then it just prints a warning.
+crictl_pull() {
+	local img=${1:-}
+	if ! command -v foo &>/dev/null; then
```

Comment:
`command -v foo`? :-)

Reply:
Good catch @fidencio ! Test code leaked to production :)

Reply:
Done